src/Resources/Shaders/BasicFragmentShader_Apple.glsl

```diff
@@ -0,0 +1,8 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
+#version 330
```

Comment:
Since we use OGL 4.1 Core _and we're on Apple hardware_, GLSL version 300 ES is not supported - we move over to version 330 to maintain compatibility (specifically in the Apple-designated shaders only).

Reply:
Makes sense. Not sure I understand their OGL support much, Mac is weird.

Reply:
Digging thru some OGL docs, it appears that Version 3.0 is specific to ES and falls between Version 3.30 (supported by OpenGL 4.1) and version 1.40 (Supported by OpenGL 3.2). Very weird indeed.